cmd/command.go

```diff
@@ -145,6 +145,87 @@ func setex(cli *Client, args [][]byte) (interface{}, error) {
 	return redcon.SimpleString(resultOK), nil
 }
 
+func setNX(cli *Client, args [][]byte) (interface{}, error) {
+	if len(args) != 2 {
+		return nil, newWrongNumOfArgsError("setnx")
+	}
+	key, value := args[0], args[1]
+	err := cli.db.SetNX(key, value)
+	if err != nil {
+		return nil, err
+	}
+	return redcon.SimpleString(resultOK), nil
+}
+
+func mSet(cli *Client, args [][]byte) (interface{}, error) {
+	if len(args) == 0 || len(args)%2 != 0 {
+		return nil, newWrongNumOfArgsError("mset")
+	}
+	err := cli.db.MSet(args...)
+	if err != nil {
+		return nil, err
+	}
+	return redcon.SimpleString(resultOK), nil
+}
+
+func mSetNX(cli *Client, args [][]byte) (interface{}, error) {
+	if len(args) == 0 || len(args)%2 != 0 {
+		return nil, newWrongNumOfArgsError("msetnx")
+	}
+	err := cli.db.MSetNX(args...)
+	if err != nil {
+		return nil, err
+	}
+	return redcon.SimpleString(resultOK), nil
+}
+
+func decr(cli *Client, args [][]byte) (interface{}, error) {
+	if len(args) != 1 {
+		return nil, newWrongNumOfArgsError("decr")
+	}
+	key := args[0]
+	return cli.db.Decr(key)
+}
+
+func decrBy(cli *Client, args [][]byte) (interface{}, error) {
+	if len(args) != 2 {
+		return nil, newWrongNumOfArgsError("decrby")
+	}
+	key, decrVal := args[0], args[1]
+	decrInt64Val, err := strconv.ParseInt(string(decrVal), 10, 64)
+	if err != nil {
```

Comment:
1. add a func in `util/str.go`, like `StrToInt64`
2. this err should not be returned directly, because it maybe a invalid number, return like this "ERR value is not an integer or out of range"